src/main.rs

```diff
@@ -387,6 +397,7 @@ mod arg_parse_tests {
                 no_default_features: false,
                 features: None,
                 manifest_path: None,
+                target: None,
```

Comment:
Can you add a test that exercises this new option?

Reply:
Added a test for this option.